src/apps/vpn/appfeaturelist.h

```diff
@@ -2,21 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-// 1. Define the FEATURE macro
-// 2. include this file
-// 3. undefine the FEATURE macro
-
-// If you want to use the callback, include `featurelistcallback.h`
-
-#if defined(_MSVC_TRADITIONAL) && _MSVC_TRADITIONAL
-#  error No supported
-#endif
-
-#define FEATURE_SIMPLE(id, name, releaseVersion, flippableOn, flippableOff, \
-                       dependencies, callback)                              \
-  FEATURE(id, name, false, L18nStrings::Empty, L18nStrings::Empty,          \
-          L18nStrings::Empty, "", "", "", releaseVersion, flippableOn,      \
-          flippableOff, dependencies, callback)
+// NOTE! This file is not meant to be used directly. Please use featurelist.h
```

Comment:
this feels confusing...like the other similar pattern...but i'm happy to wait and see i'm probably missing context.

Reply:
Similar pattern. I update the comment here as well.